.vscode/settings.json

```diff
@@ -2,5 +2,10 @@
   "files.exclude": {
     "**/.git": true,
     "**/coverage": true
-  }
+  },
+  "eslint.options": {
+    "configFile": "/Users/mmaciorowski/Documents/Projekty/JS/carbon/.eslintrc"
```

Comment:
this is linking to your personal workspace

Reply:
my bad, sorry